### DISCUSSION TITLE
Welcome to NotionNext Discussions!

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tangly1024** [comment]
Hi👋🏻，我是 tangly1024 - NotionNext项目的作者!

欢迎大家畅所欲言 :)


  Telegram : https://t.me/+sOvnlf2QKZQ3NDU1

**Comment by chancenju** [comment]
居然没人讨论。求作者介绍一下如何开启看板 pet

**Reply by tangly1024** [reply]
😂讨论功能才刚刚开启～🎉欢迎第一位访客🎉

- 看板PET目前只支持NEXT主题，配置方式：
[/themes/NEXT/config_next.js](https://github.com/tangly1024/NotionNext/blob/main/themes/NEXT/config_next.js
)中修改：`WIDGET_PET` 的值为`true`即可～
```
  WIDGET_PET: false, // 是否显示宠物挂件
  WIDGET_PET_LINK: 'https://cdn.jsdelivr.net/npm/live2d-widget-model-wanko@1.0.5/assets/wanko.model.json', // 挂件模型地址 @see https://github.com/xiazeyu/live2d-widget-models
 ```

**Comment by chancenju** [comment]
thank you.刚开始尝试搭建，在nextstarter，nobelium和你这个二次开发中选择了你哈哈哈，就是想除了blog之外能多几个栏目，放放video啥的。现在的page不知道放哪里显示。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年2月23日(星期三) 下午5:38
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [tangly1024/NotionNext] Welcome to NotionNext Discussions! (Discussion #59)





 
😂讨论功能才刚刚开启～🎉欢迎第一位访客🎉
  
看板PET目前只支持NEXT主题，配置方式：
 /themes/NEXT/config_next.js中修改：WIDGET_PET 的值为true即可～
    WIDGET_PET: false, // 是否显示宠物挂件   WIDGET_PET_LINK: ***@***.***/assets/wanko.model.json', // 挂件模型地址 @see https://github.com/xiazeyu/live2d-widget-models  
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by tangly1024** [reply]
我是做纯粹的博文记录，目前没想过增加别的栏目😝，非post的单页，只能是类似[about](https://tangly1024.com/about)这样的简单页面。

你指的新的栏目是什么样的呢，假设是多个video，可能要单独设计一个页面来展示列表了🤣

类似这样的配置文件，当类型是视频时，访问指定的path会展现不同的模块内容
```
  "PageConfig" :[
  {"id":"xxx[notionPageId]","type":"post","path":"blog"}, // 博文列表
  {"id":"xxx[notionPageId]","type":"video","path":"vlog"}, // 视频板块
  {"id":"xxx[notionPageId]","type":"discussion","path":"discussion"}, // 留言
 ]
```

**Comment by chancenju** [comment]
可能就是想多几个单页。能指导一下如何实现吗，就是想多几个类似about这样的简单页面，resume，photo library之类的。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "tangly1024/NotionNext"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月23日(星期三) 晚上6:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [tangly1024/NotionNext] Welcome to NotionNext Discussions! (Discussion #59)





 
我是做纯粹的博文记录，目前没想过增加别的栏目😝，非post的单页，只能是类似about这样的简单页面。
 
你指的新的栏目是什么样的呢，假设是多个video，可能要单独设计一个页面来展示列表了🤣
 
类似这样的配置文件，当类型是视频时，访问指定的path会展现不同的模块内容
   "PageConfig" :[   {"id":"xxx[notionPageId]","type":"post","path":"blog"}, // 博文列表   {"id":"xxx[notionPageId]","type":"video","path":"vlog"}, // 视频板块   {"id":"xxx[notionPageId]","type":"discussion","path":"discussion"}, // 留言  ]  
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by tangly1024** [reply]
要添加新的单页可以参考[这篇内容](https://github.com/tangly1024/NotionNext/issues/38)的做法；不过这样做需要改动到代码，有一定技术的门槛。

另一个好消息是，我觉得你的这个需求提议很有趣，我正计划开发一个新功能😂.
【快速添加单页】
Notion中当文章的类型为“Page”，则被视为新的单页栏目，被自动展示添加到博客的导航菜单栏～

**Reply by tangly1024** [reply]
项目已升级至V2.5.0版本， 新增功能 [添加单页](https://docs.tangly1024.com/zh/features/singlePage)
需要更新至最新版本使用～
https://github.com/tangly1024/NotionNext/releases/tag/v2.5.0

**Comment by luoliguang** [comment]
怎么用Telegram呀？我之前能登录，但是现在收不到验证码了
